documentation/STRUCTURE.md

```diff
@@ -0,0 +1,161 @@
+# Documentation
+
+For the purpose of this proposal document we consider all of the following as documentation:
+
+- documentation in our code within the Fairlearn repository which is currently also hosted on readthedocs
+- Jupyter notebooks
+- project website
+- blog posts
+
+Throughout the document we compare with the documentation of other popular projects such as 
+
+- [scikit-learn](https://scikit-learn.org/)
+- [pandas](https://pandas.pydata.org/)
+
+## Goals
+
+The documentation should be
+
+- discoverable, ideally in a single place as opposed to multiple
+- clear
+- concise when describing individual pieces of functionality
+- detailed when describing entire application scenarios, e.g. in the form of case studies
+- available for the latest version, but if possible also for past versions ([example](https://scikit-learn.org/dev/versions.html))
+- maintainable: it should be simple for maintainers to update/validate
+- without ads (readthedocs always has ads that are shown alongside our documentation)
+
+## Proposal
+
+Like for most projects the website, [fairlearn.org](http://fairlearn.org) will be the central place to look for documentation.
+From there visitors have various paths to explore content depending on what they are looking for, as detailed in the following subsections.
+
+### Homepage
+
+```
+website
+|--- Getting Started
+|--- API reference
+|--- User guide
+|--- Case studies
+|--- Community
+```
+
+### Getting Started
+
+This page provides information on
+- Installation
+- terminology (or at least a link to it)
+- where to go next, e.g. tutorials, videos, articles
+
+Installation should cover various platforms, which should be very straightforward. Any reoccurring patterns in reported issues should be listed, as well as how to troubleshoot them. It may end up similar to [this guide](https://scikit-learn.org/dev/install.html)).
+
+Example:
+
+- [pandas](https://pandas.pydata.org/getting_started.html)
+- [scikit-learn](https://scikit-learn.org/dev/getting_started.html)
+
+### API reference
```

Comment:
Something to be decided very soon - do we change to numpy docstrings?

Reply:
It's in the "less urgent" (read: lower priority) category because it doesn't prevent us from creating the website, but if there's time I want to do it.

Reply:
resolving since that's included in the doc